assets/prototype/application/hooks/useZonedTime.ts

```diff
@@ -0,0 +1,21 @@
+import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz';
+import useConfig from '../../application/services/config/useConfig';
+
+const useZonedTime = () => {
+	const { timezone } = useConfig();
+
+	const zonedTimeToUtcCb = (date: Date | string | number) => {
+		return zonedTimeToUtc(date, timezone);
+	};
+
+	const utcToZonedTimeCb = (isoDate: Date | string | number) => {
```

Comment:
This parameter name is used as seen here:
![image](https://user-images.githubusercontent.com/1477453/70900942-7f8a1280-2002-11ea-8992-9f25b2542af6.png)
***
[link](https://github.com/marnusw/date-fns-tz#utctozonedtime)